### DISCUSSION TITLE
Example to show how to use Axum's middlewares with Leptos routes?

### DISCUSSION THREAD

**Discussion by sergeda** [leptos-rs/leptos#2600] [Q&A]
I'm using Axum's middlewares that provides some dependencies to the routes.
Is there any examples how to combine them with Leptos routes?

**Comment by gbj** [comment]
Some more details may be helpful. What have you tried? What hasn't worked? You say "some dependencies" -- does that mean you're providing something you'd normally extract with an extractor? etc.
